src/main/resources/temporary_study.cql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS study.temporaryStudy (
```

Comment:
A general comment: is 'temporary' the right term? It is not really temporay because it does not mode a study but an ongoing study creation.

Reply:
Indeed it is not the right term
Perhaps 'ongoingCreationStudy' ?'

Reply:
StudyCreationRequest ?

Reply:
StudyCreationRequest c'est bien

Reply:
ok